docs/install_spock.md

```
@@ -76,33 +76,14 @@ other Postgres extension:
 ## Configuring Postgres for Spock Replication
 
 After installing Postgres on each node that will host a spock instance,
-use [initdb](https://www.postgresql.org/docs/18/app-initdb.html) to initialize
+use [initdb](https://www.postgresql.org/docs/current/app-initdb.html) to initialize
 a Postgres cluster. After initializing the cluster, you can connect
 with psql, and query the server for file locations and cluster details:
 
 ```sql
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)